ts/Grid/Pro/GridEvents.ts

```diff
@@ -207,7 +210,25 @@ export interface CellEvents {
  */
 export interface ColumnEvents {
     /**
-     * Callback function to be called when the column is sorted for instance,
+     * Callback function to be called when the column is filtered,
+     * before clicking on header.
+     */
+    beforeFiltering?: ColumnEventCallback;
+
+    /**
+     * Callback function to be called when the column is filtered,
+     * after clicking on header.
```

Comment:
`before/after clicking on header` is that right? I don't think so...
It's either on keypress in the input or on the  dropdown change